website/views/options/social/index.jade

```diff
@@ -113,3 +113,17 @@ script(type='text/ng-template', id='partials/options.social.html')
   .tab-content
     .tab-pane.active
       div(ui-view)
+
+script(type='text/ng-template', id='modals/copyChatToDo.html')
+  .modal-header
+    h4=env.t('copyMessageAsToDo')
+  .modal-body
+    p
```

Comment:
These should be `.form-group` instead of `p`, then you won't need the `br`

```
    .form-group
      input.form-control(type='text',ng-model='text')
    .form-group
      textarea.form-control(rows='5',ng-model='notes', focus-me)
```